test/tools/TestService/Program.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace TestService
 {
-    static class Program
+    internal static class Program
     {
-       static void Main()
+       private static void Main()
```

Comment:
I wonder why they are not public.

Reply:
This PR adds the default accessibility modifiers: `internal` for class and `private` for method.

It seems the class has no outside callers therefore accessibility is not important:

https://stackoverflow.com/questions/3110184/why-is-main-method-private